tests/unit/challenges/test_urls.py

```diff
@@ -0,0 +1,77 @@
+from django.urls import reverse_lazy
+from rest_framework import status
+from rest_framework.test import APITestCase, APIClient
+from challenges.models import Challenge, ChallengePhase
+from participants.models import Participant, ParticipantTeam
+from hosts.models import ChallengeHost, ChallengeHostTeam
+from base.models import TimeStampedModel
+from django.contrib.auth.models import User
+import unittest
```

Comment:
can you please reorder your imports according to the following list
* python packages(here unittest)
* django packages(here django.urls)
* thirdy party django packages(here rest_framework)
* project import like apps etc(here challenges, participants, etc)

Also leave a blank line after each set of group and make sure that these imports are written alphabetically.